playbooks/roles/validation/tasks/main.yml

```diff
@@ -13,3 +13,13 @@
   fail:
     msg: "Too many VPN clients specified. vpn_clients must be greater than 0 and less than 20"
   when: (vpn_clients > 20) or (vpn_clients < 1)
+
+- name: Validate that at least one VPN is specified
+  fail:
+    msg: "At least one VPN service must be installed :("
```

Comment:
I think this `msg` might cause some confusion. I could imagine a confused user thinking they needed to install one of these VPN services on the Streisand builder machine.

How about: "At least one Streisand VPN service must be enabled".